Library/Homebrew/cask/lib/hbc/cask.rb

```diff
@@ -27,7 +27,7 @@ def metadata_master_container_path
     end
 
     def metadata_versioned_container_path
-      cask_version = version ? version : :unknown
+      cask_version = version ? version.sanitize : :unknown
```

Comment:
Let's move the `version.sanitize` stuff to a separate PR since it's not necessary to load the cask file from the `.metadata` directory.